src/execution_plan/ops/op_merge.c

```diff
@@ -102,21 +102,22 @@ static void _CommitEdges(OpMerge *op, Record r) {
 	if(op->stats) op->stats->relationships_created += edge_count;
 }
 
-static void _CreateEntities(OpMerge *op, Record r) {
+static bool _CreateEntities(OpMerge *op, Record r) {
```

Comment:
Same comment as for op_create

Reply:
we are issuing a runtime error, at `query_ctx.c`
this is the fallback incase the runtime error falis.